### PR TITLE
UI Changes

### PR DESCRIPTION
**Pull request by Charchit507**

### CODE REVIEW

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Users::SessionsController < Devise::SessionsController
```
Comment (neeteshgupta95) [inline app/controllers/users/sessions_controller.rb]: remove that file

File: app/controllers/users/registrations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  # before_action :configure_sign_up_params, only: [:create]
```
Comment (neeteshgupta95) [inline app/controllers/users/registrations_controller.rb]: remove that file

File: app/controllers/users/passwords_controller.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Users::PasswordsController < Devise::PasswordsController
+  # GET /resource/password/new
+  # def new
```
Comment (neeteshgupta95) [inline app/controllers/users/passwords_controller.rb]: remove that file

File: app/controllers/users/confirmations_controller.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
+  # def new
```
Comment (neeteshgupta95) [inline app/controllers/users/confirmations_controller.rb]: remove that file

File: app/controllers/users/unlocks_controller.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::UnlocksController < Devise::UnlocksController
+  # GET /resource/unlock/new
```
Comment (neeteshgupta95) [inline app/controllers/users/unlocks_controller.rb]: remove that file

**Review by neeteshgupta95 — CHANGES_REQUESTED** [review]

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  # You should configure your model like this:
```
Comment (neeteshgupta95) [inline app/controllers/users/omniauth_callbacks_controller.rb]: remove that file

File: app/controllers/users/sessions_controller.rb [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  # before_action :configure_sign_in_params, only: [:create]
```
Comment (neeteshgupta95) [inline app/controllers/users/sessions_controller.rb]: remove that file

File: db/migrate/20220816065006_changes_in_tables.rb [hunk 8]
```diff
@@ -0,0 +1,6 @@
+class ChangesInTables < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
+  end
+end
```
Comment (neeteshgupta95) [inline db/migrate/20220816065006_changes_in_tables.rb]: why did you add those things in this PR?

**Review by neeteshgupta95 — CHANGES_REQUESTED** [review]

File: db/migrate/20220816065006_changes_in_tables.rb (hunk 8 above)
Comment (Charchit507) [reply db/migrate/20220816065006_changes_in_tables.rb]: sir mistake ho gyi

Comment (neeteshgupta95) [reply db/migrate/20220816065006_changes_in_tables.rb]: remove those changes

File: db/schema.rb [hunk 9]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_08_10_064022) do
+ActiveRecord::Schema.define(version: 2022_08_16_083515) do
```
Comment (neeteshgupta95) [inline db/schema.rb]: remove that change

File: app/views/layouts/application.html.erb [hunk 10]
```diff
@@ -10,8 +10,13 @@
   </head>
 
   <body>
-    <p class="notice"><%= notice %></p>
-    <p class="alert" style="padding:0rem"><%= alert %></p>
-    <%= yield %>
+    <%= render "header_&_footer/header" %>
```
Comment (neeteshgupta95) [inline app/views/layouts/application.html.erb]: rename it from `header_&_footer `to `header_and_footer`

File: db/schema.rb (hunk 9 above)
Comment (Charchit507) [reply db/schema.rb]: sir  2022_08_10_064022 yhi edit krna hai kya

**Review by neeteshgupta95 — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 9 above)
Comment (neeteshgupta95) [reply db/schema.rb]: why its showing any new changes?

Comment (Charchit507) [reply db/schema.rb]: i mean day, hour and minutes

Comment (neeteshgupta95) [reply db/schema.rb]: why changes are coming in that file?

**Review by neeteshgupta95 — APPROVED** [review]
